src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -1168,7 +1190,8 @@ public new int[] Context
 
         private int _postContext = 0;
 
-        private IContextTracker GetContextTracker() => (_preContext == 0 && _postContext == 0) ? _noContextTracker : new ContextTracker(_preContext, _postContext);
+        // When we are in Raw mode or pre- and postcontext are zero, use the _noContextTracker, since we will not be needing trackedLines.
+        private IContextTracker GetContextTracker() => (Raw || (_preContext == 0 && _postContext == 0)) ? _noContextTracker : new ContextTracker(_preContext, _postContext);
```

Comment:
Since we're adding something here anyway, I feel it might be time to tidy this long line up a bit?

```suggestion
        private IContextTracker GetContextTracker() => (Raw || (_preContext == 0 && _postContext == 0)) ? _noContextTracker : new ContextTracker(_preContext, _postContext);
```
```suggestion
        private IContextTracker GetContextTracker() => (Raw || (_preContext == 0 && _postContext == 0)) 
            ? _noContextTracker
            : new ContextTracker(_preContext, _postContext);
```

Reply:
Agreed, fixed.